src/thor/bidirectional_astar.cc

```diff
@@ -512,9 +516,12 @@ inline bool BidirectionalAStar::ExpandReverseInner(GraphReader& graphreader,
   // Get cost. Use opposing edge for EdgeCost. Separate the transition seconds so we
   // can properly recover elapsed time on the reverse path.
   const Cost transition_cost =
-      costing_->TransitionCostReverse(meta.edge->localedgeidx(), nodeinfo, opp_edge, opp_pred_edge);
-  const Cost newcost =
-      pred.cost() + costing_->EdgeCost(opp_edge, t2, time_info.second_of_week) + transition_cost;
+      costing_->TransitionCostReverse(meta.edge->localedgeidx(), nodeinfo, opp_edge, opp_pred_edge,
+                                      pred.has_flow_speed());
+  uint8_t flow_sources;
+  const Cost newcost = pred.cost() +
```

Comment:
Added flow_sources to TransitionCostReverse so that we can save it off in the edge label.